Cod3rsGrowth.forms/FormEditarCarro.cs

```diff
@@ -69,22 +59,22 @@ private void AoClicarNoBotaoSalvarEdicaoCarro(object sender, EventArgs e)
                 };
 
                 _servicoCarro.Editar(carroEditado);
-                MessageBox.Show($"Carro com ID {_idDaEdicao} editado com sucesso!", "Editando carro");
-                this.Close();
+                MessageBox.Show($"Carro com ID {_carro.Id} editado com sucesso!", "Editando carro");
```

Comment:
Quando essa linha for executada o carro já vai ter sido editado. Então a mensagem tem que demonstrar isso, tanto a mensagem quanto o titulo dela.